contracts/multisig/src/contract.rs

```diff
@@ -137,12 +138,19 @@ impl Multisig {
             title: title.clone(),
             description,
         };
+
+        let creation_timestamp = env.ledger().timestamp();
+        let expiration_timestamp = creation_timestamp + deadline.unwrap_or(SEVEN_DAYS_DEADLINE);
```

Comment:
Add some check that the deadline is not earlier or equal to the time right now, it must be longer then 1 hour (just for the sake if someone doesn't get timestamp format and puts 1 or something). Return an error if not.

Reply:
`expiration_date` wlil always be over `creation_timestamp`, so I'm checking if we are at least an hour on top of that.

Reply:
But this is wrong then, it should be
```suggestion
        let expiration_timestamp = deadline.unwrap_or(creation_timestamp +SEVEN_DAYS_DEADLINE);
        if expiration_timestamp <= creation_timestamp {
            return Err()
```
And rename this  parameter from `deadline` to `expiration_date`.